shell/key-bindings.zsh

```diff
@@ -112,7 +112,7 @@ fzf-history-widget() {
   # history lines, is loaded, and that Perl is installed for multi-line output.
   if zmodload -F zsh/parameter p:history 2>/dev/null && (( ${#commands[perl]} )); then
     selected="$(printf '%1$s\t%2$s\000' "${(vk)history[@]}" |
-      perl -0 -ne 'if (!$seen{(/^\s*[0-9]+\**\s+(.*)/, $1)}++) { s/\n/\n\t/gm; print; }' |
+      perl -0 -ne 'if (!$seen{(/^\s*[0-9]+\**\s+((?s).*)/, $1)}++) { s/\n/\n\t/gm; print; }' |
```

Comment:
Thanks, I realized that I had a vague understanding of these flags.

Can we put `s` flag globally? It would result in a shorter code. Also, it looks like we can drop `m` from the latter regex.

```suggestion
      perl -0 -ne 'if (!$seen{(/^\s*[0-9]+\**\s+(.*)/s, $1)}++) { s/\n/\n\t/g; print; }' |
```